extensions/skins/conversation_v1/Conversation.js

```diff
@@ -285,6 +286,28 @@ oppia.directive('conversationSkin', [function() {
         }
       };
 
+      $scope.readSelectedTextAloud = function() {
+        var content = $scope.activeCard.contentHtml;
+        var contentElement = document.createElement('div');
+        contentElement.innerHTML = content;
```

Comment:
I want us to not use innerHTML in the frontend, since it can lead to security issues. (I don't think it will in this particular case, but I'd prefer not to set a precedent.)

Do you think it would be fine to interpolate the string for all <oppia-noninteractive-*> tags, replace them with their textual equivalents, then take the .textContent() of the entire thing and read that instead?